fiftyone/core/collections.py

```diff
@@ -9043,6 +9046,14 @@ def get_index_information(self):
         # Sample-level indexes
         fields_map = self._get_db_fields_map(reverse=True)
         sample_info = self._dataset._sample_collection.index_information()
+
+        if include_size:
+            dataset_stats = self._dataset.stats(include_indexes=True)
```

Comment:
It is not ideal to do two calls to MongoDB to get the stats ~ still trying to figure out a better way :)